tests/test_github_sdk.py

```diff
@@ -13,6 +14,11 @@
 DSN = "https://foo@random.ingest.sentry.io/bar"
 TOKEN = "irrelevant"
 
+# Different versions of Python represent certain errors differently
```

Comment:
I noticed this because my venv on another computer got set up with Python 3.8 rather than 3.10.